pylint/checkers/typecheck.py

```diff
@@ -1294,16 +1294,23 @@ def visit_call(self, node: nodes.Call) -> None:
         # Ignore instances of descriptors since astroid cannot properly handle them
         # yet
         if called and not called.callable():
```

Comment:
Maybe we can create a function from line 1296 to line 1318 and use premature return to simplify this expression ?

```python
 if not isinstance(called, astroid.Instance):
    self._check_uninferable_call(node)
    return
if not has_known_bases(called):
    return
....        
```